trunk/NDHMS/Land_models/NoahMP/IO_code/Makefile

```diff
@@ -17,7 +17,7 @@ all:	$(OBJS_NoahMP) $(OBJS)
 
 NoahMP : $(OBJS_NoahMP) $(OBJS)
 
-module_NoahMP_hrldas_driver.o: module_NoahMP_hrldas_driver.F ../../../HYDRO_drv/module_HYDRO_drv.o  ../../../Data_Rec/module_namelist.o ../../../Data_Rec/module_RT_data.o ../../../CPL/NoahMP_cpl/module_hrldas_HYDRO.o
+module_NoahMP_hrldas_driver.o: module_NoahMP_hrldas_driver.F ../../../HYDRO_drv/module_HYDRO_drv.o  ../../../Data_Rec/module_namelist.o ../../../Data_Rec/module_RT_data.o
```

Comment:
why is this dependence removed?

Reply:
We can't remember why we removed it. I am putting it back in.

Reply:
If we dont need it, it's probably better to take it out. 
I just thought there might be a clean conceptual explanation for its removal.